### PR TITLE
chore - fix audit issues

### PR DESCRIPTION
**Pull request by devinea**
- `tough-cookie` - https://github.com/advisories/GHSA-72xf-g2v4-qvf3
- `word-wrap` - https://github.com/advisories/GHSA-j8xg-fqg3-53r7
Update `yeoman-generator` in `@sap-ux/generator-simple-fe`

### CODE REVIEW

**Review by zdravko-georgiev — APPROVED** [review]
Thanks @devinea

File: docs/version-overrides.md [hunk 1]
```diff
@@ -44,3 +66,5 @@ This document lists the version overrides for vulnerable (nested) dependencies a
 :warning: Attention :warning: 
 * `trim`, `trim-newlines` and `glob-parent` are dependencies of `storybook` that is used in `@sap-ux/ui-components`. Once a new version without the vulnerable dependency is available, it is to be used and this override can be removed.
 * `semver` is used by too many modules. Override can be removed or kept for specific dependencies after we have cleaned up our devDependencies.
+* `tough-cookie` no fix available yet for jest-environment-jsdom
+*  `word-wrap` no fix available yet for @typescript-eslint/* modules.
```
Comment (donal-tobin-sap) [inline docs/version-overrides.md:70]: Fixed with fork https://www.npmjs.com/package/@aashutoshrathi/word-wrap  version 1.2.5+

File: docs/version-overrides.md [hunk 2]
```diff
@@ -44,3 +66,5 @@ This document lists the version overrides for vulnerable (nested) dependencies a
 :warning: Attention :warning: 
 * `trim`, `trim-newlines` and `glob-parent` are dependencies of `storybook` that is used in `@sap-ux/ui-components`. Once a new version without the vulnerable dependency is available, it is to be used and this override can be removed.
 * `semver` is used by too many modules. Override can be removed or kept for specific dependencies after we have cleaned up our devDependencies.
+* `tough-cookie` no fix available yet for jest-environment-jsdom
```
Comment (donal-tobin-sap) [inline docs/version-overrides.md:69]: No requirement to add an override only need to rebuild the lock file as the dependencies  to tough cookie are all open ranges.

**Review by donal-tobin-sap — CHANGES_REQUESTED** [review]
There is a fix for word-wrap.

File: docs/version-overrides.md (hunk 1 above)
Comment (donal-tobin-sap) [reply docs/version-overrides.md:58]: Fork is no longer required. (released 1 week ago - original finding is near 3 weeks old)
There is an official fix now https://github.com/jonschlinkert/word-wrap/releases/tag/1.2.5
Note that the 1.2.5 version should be used to not get space changes.

**Review by donal-tobin-sap — APPROVED** [review]
Looks good for new word-wrap release.

**Review by zdravko-georgiev — APPROVED** [review]

**Comment by devinea** [comment]
Thanks @donal-tobin-sap @zdravko-georgiev

**Pull request merged**